### ISSUE TITLE
intellij support

### ISSUE THREAD

**Issue by kpbochenek** [etorreborre/specs2#481]
Hi, I encounter a few annoying issues with specs2 under intellij.
This is not strictly specs2 bug so if you think I should move this topic somewhere else let me know.

I have a bunch of classes with tests inheriting from `org.specs2.mutable.SpecificationWithJUnit`
It is a maven based project.
Intellij version 2016.1

When I right click in an open test file to select 'Run XXXTest' it takes many seconds for a menu to open.
It is probably because implicit resolution?

Another annoying thing is I can't run a single test from my specification. When I right click on it I can only see a choice to run whole specification.

I am using form:
```
"TestActor" should {

    "do something on received message" in new TestActorScope {
```

Can I change something in settings / refactor my tests to be more intellij friendly?

Thanks in advance!

**Comment by etorreborre** [comment]
I'll have a look at this later on today. 

About selecting just one test to run, this used to work with Intellij but I know it doesn't anymore. You can work around this by selecting the class to execute and passing `-Dex="name or part of the name of your test"`.

**Comment by etorreborre** [comment]
I just tested the selection of a single test and it works for me:
![image](https://cloud.githubusercontent.com/assets/10988/15401522/35b25d60-1df0-11e6-8187-b129ff8a64ff.png)

However it takes a few seconds before appearing which is annoying. Can you please open an issue on the [Intellij bug tracker](https://youtrack.jetbrains.com/issues/SCL) for this?

**Comment by kpbochenek** [comment]
Okay I created an issue: https://youtrack.jetbrains.com/issue/SCL-10302

Nice I initially used JUnit configuration but when I choose configuration "Specs2" it works for single test.

Because I inherited my class from SpecificationWithJUnit I though this would be supported by "JUnit" runner but it is best to use "Specs2" version.
It is bad "Specs2" configuration doesn't have "Repeat Until Failure" which JUnit has but I should also report that in intellij bug tracker.

Thanks for help :)
